test/integration/scheduler/framework_test.go

```diff
@@ -21,57 +21,46 @@ import (
 	"testing"
 	"time"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit: remove the added "v1". The problem is usually that the editor you are using relies on goimports rather than gofmt to do formatting.

Reply:
I kept asking people to remove these aliases, but now I feel it is not worth fighting the tool. goimport does this and many PRs have this king of aliasing.

Reply:
FYI: to eliminate the `v1` in editor settings (e.g. vscode): https://groups.google.com/d/msg/kubernetes-dev/Y2ShVRrU4xM/LKfaiFqAAwAJ

Reply:
> FYI: to eliminate the `v1` in editor settings (e.g. vscode): https://groups.google.com/d/msg/kubernetes-dev/Y2ShVRrU4xM/LKfaiFqAAwAJ

Thanks for the information, but I used emacs for development, this does not work for me 😢

However, I do find my way to solve this problem through [magit](https://magit.vc/) which could help me to commit changes by chunk, and these changes are accidentally committed. 🐱